package androidx.compose.material.icons.filled;

import bj.n;
import g1.m0;
import g1.t;
import j0.a;
import k1.e;
import k1.f;
import k1.h0;
import kotlin.jvm.internal.l;

/* JADX INFO: compiled from: r8-map-id-cf66f36031e302b9a7b5a76da57d208ae1b6ba4a38bba3bee1d4ed6983c54df3 */
/* JADX INFO: loaded from: classes.dex */
public final class ElectricBoltKt {
    private static f _electricBolt;

    public static final f getElectricBolt(a aVar) {
        l.f("<this>", aVar);
        f fVar = _electricBolt;
        if (fVar != null) {
            return fVar;
        }
        e eVar = new e("Filled.ElectricBolt", 24.0f, 24.0f, 24.0f, 24.0f, 0L, 0, false, 224);
        int i10 = h0.f9067a;
        m0 m0Var = new m0(t.f6905b);
        n nVarC = k0.a.c(14.69f, 2.21f, 4.33f, 11.49f);
        nVarC.i(-0.64f, 0.58f, -0.28f, 1.65f, 0.58f, 1.73f);
        nVarC.l(13.0f, 14.0f);
        nVarC.m(-4.85f, 6.76f);
        nVarC.i(-0.22f, 0.31f, -0.19f, 0.74f, 0.08f, 1.01f);
        nVarC.k(0.0f);
        nVarC.i(0.3f, 0.3f, 0.77f, 0.31f, 1.08f, 0.02f);
        nVarC.m(10.36f, -9.28f);
        nVarC.i(0.64f, -0.58f, 0.28f, -1.65f, -0.58f, -1.73f);
        nVarC.l(11.0f, 10.0f);
        nVarC.m(4.85f, -6.76f);
        nVarC.i(0.22f, -0.31f, 0.19f, -0.74f, -0.08f, -1.01f);
        nVarC.m(0.0f, 0.0f);
        nVarC.h(15.47f, 1.93f, 15.0f, 1.92f, 14.69f, 2.21f);
        nVarC.g();
        e.a(eVar, nVarC.f3166a, 0, m0Var);
        f fVarB = eVar.b();
        _electricBolt = fVarB;
        return fVarB;
    }
}
